crates/binstalk-downloader/Cargo.toml

```
@@ -78,7 +80,7 @@ native-tls = [
 ]
 
 # Enable trust-dns-resolver so that features on it will also be enabled.
-trust-dns = ["trust-dns-resolver", "reqwest/trust-dns"]
+trust-dns = ["trust-dns-resolver", "reqwest/trust-dns", "default-net", "ipconfig"]
```

At this point we can stop enabling `reqwest/trust-dns` since we no longer use it.

```suggestion
trust-dns = ["trust-dns-resolver", "default-net", "ipconfig"]
```